makefile

```diff
@@ -128,3 +89,57 @@ serve-frontend:
 
 serve-backend:
 	uvicorn backend.app:APP --reload
+
+# Help
+help:
```

Comment:
## `make help`
Running this will now show show each command and what it does.

I wanted to do this in a programmatic way; to just list out all the commands. But I couldn't find a way that would work.